skia/RSkSVGNode.cpp

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright (C) 1994-2023 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "ReactSkia/utils/RnsLog.h"
+#include "RSkSVGContainer.h"
+#include "RSkSVGNode.h"
+
+namespace facebook {
+namespace react {
+
+RSkSVGNode::RSkSVGNode(SkSVGTag tag) : INHERITED(tag) {};
+
+void RSkSVGNode::setCommonRenderableProps(const RNSVGCommonRenderableProps  &renderableProps) {
+
+  RNS_LOG_DEBUG("Supported Properties count  :" << renderableProps.propList.size() << " : for SVG TAG: "<< (int)tag());
+  RNS_LOG_DEBUG("##################");
+  for (auto props : renderableProps.propList ) {
+    RNS_LOG_DEBUG(props);
+  }
+  RNS_LOG_DEBUG("##################");
+
+  for (auto &item : renderableProps.propList) {
+    //Fill Props
```

Comment:
to avoid the multiple if-else , see if there is possibility to optimize further 
Example reference - https://github.com/nagra-opentv/react-native-reanimated/blob/wip/rns/skia/Nodes/REAOpNode.cpp